gemrb/GUIScripts/GUISTORE.py

```diff
@@ -206,7 +205,6 @@ def ChangeStoreView(func):
 	Store = GemRB.GetStore ()
 	#based on shop type, these buttons will change
 	store_buttons = Store['StoreButtons']
-	BarteringPC = GemRB.GameGetFirstSelectedPC ()
```

Comment:
This is where it changes now. If you switch the pc and then the store window, eg. from stealing to identification, it would pick up the new pc as the barterer (unless this branches changes to GameGetFirstSelectedPC changed it).

Reply:
Ah no, just a diff header issue.